Bottom: 9160094d2e198f4c0bb6d55f1e70c765d6849267
Top:    aa7e572508232d1f27cdc1a84a2070cc1f9b7832
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-07 00:58:53 +0000

find.c: The User varaible needs to die...

Reduce its usage to where absolutely necessary... oh, what a surprise,
another variable passing state down into add_object_to_list ()


---

diff --git a/src/find.c b/src/find.c
index 1ef631b..3bd66f6 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3093,7 +3093,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
    */
   ListStart (type, ptr1, ptr2, ptr3, flag);
   DoIt (flag, AndRats, AndDraw);
-  if (User)
+  if (AndDraw)
     IncrementUndoSerialNumber ();
   User = false;
